addons/website_mass_mailing/views/snippets_templates.xml

```diff
@@ -59,7 +59,27 @@ database, without the s_newsletter_list class. See fixNewsletterListClass.
             <form id="newsletter_form" action="/website/form/" method="post" enctype="multipart/form-data" class="o_mark_required"
                   data-mark="*" data-model_name="mailing.contact" data-success-mode="message" hide-change-model="true">
                 <div class="s_website_form_rows row s_col_no_bgcolor">
-                    <div class="mb-0 py-2 col-12 s_website_form_field s_website_form_model_required" data-type="char" data-name="Field">
+                    <t t-if="request.env['mailing.contact']._is_name_split_activated()">
+                        <div class="mb-0 py-2 col-12 s_website_form_field s_website_form_model_required" data-type="char" data-name="Field">
```

Comment:
It seems like this first div is exactly the same between the 2, so maybe it should stay out of the if/else?